### PR TITLE
update MANIFEST.in, enable check manifest hook

### PR DESCRIPTION
**Pull request by Zeitsperre**
### Pull Request Checklist:
- [x] `pre-commit` hooks are installed/active in my local clone (`$ pre-commit install`)
- [ ] This PR addresses an already opened issue (for bug fixes / features)
    - This PR fixes #xyz
- [ ] (If applicable) Documentation has been added / updated (for bug fixes / features)
- [ ] HISTORY.rst has been updated (with summary of main changes)
  - [ ] Link to issue (:issue:`number`) and pull request (:pull:`number`) has been added
- [ ] `bumpversion patch` has been called on this branch

### What kind of change does this PR introduce?

* Updates manifest and enables the check manifest hook

### Does this PR introduce a breaking change?

No.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -47,10 +47,10 @@ repos:
...
       - id: yamllint
         args: ['--config-file', '.yamllint.yaml']
-#  - repo: https://github.com/mgedmin/check-manifest
-#    rev: "0.48"
-#    hooks:
-#      - id: check-manifest
+  - repo: https://github.com/mgedmin/check-manifest
+    rev: "0.48"
+    hooks:
+      - id: check-manifest
```
Comment (aulemahal) [inline .pre-commit-config.yaml:53]: https://github.com/mgedmin/check-manifest/blob/master/CHANGES.rst#048-2022-03-13

Git under 2.11 is not supported by revision 0.48 of `check-manifest`. Thus, this pre-commit hook systematically fails on doris (git 1.8.3), which is where I do most of my xscen development. Which means it affects at least 25% of the devs! ;)

Comment (Zeitsperre) [reply .pre-commit-config.yaml:53]: Right! I had forgotten about that! Have you asked Mourad if we could update git on our internal infra?

Comment (RondeauG) [reply .pre-commit-config.yaml:53]: Je viens de lui écrire.

Comment (RondeauG) [reply .pre-commit-config.yaml:53]: Maybe not, after all... See Pascal's message on Slack.

**Comment by aulemahal** [comment]
Pour des raisons qui m'échappent, git 2.37 ne fonctionne toujours pas sur doris...

Aussi, je recopie ici la description du readme de "check-manifest": 

> Are you a Python developer? Have you uploaded packages to the Python Package Index? Have you accidentally uploaded broken packages with some files missing? If so, check-manifest is for you.

Mes réponses: Yes, No, No.
Ainsi donc, je propose de mettre ce problème plus bas sur la pile. On attendra le moment où on se rend compte qu'il vaudrait mieux envoyer `xscen` sur pypi (ce qui n'est pas dans nos plans même long terme, il me semble?).

**Pull request merged**